README.rst

```diff
@@ -124,6 +124,45 @@ g. *Implementation Plan* (Optional): If accepted who will implement the change?
 
   This is also a good section to outline the changes to the implementation, or otherwise include insights that assume familiarity with the implementation.
 
+Here are some characteristics that a good proposal should have
+
+* It should be self-standing.  Some proposals accumulate a long and interesting discussion
+  thread, but in ten years time all that will be gone (except for the most assiduous readers).
+  Before acceptance, therefore, the proposal should be edited to reflect the fruits of
+  that dicussion, so that it can stand alone.
+
+* It should be precise, especially the "Proposed change specification"
+  section.  Language design is complicated, with lots of
+  interactions. It is not enough to offer a few suggestive examples
+  and hope that the reader can infer the rest.  Vague proposals waste
+  everyone's time; precision is highly valued.
+
+  We do not insist on a fully formal specification, with a
+  machine-checked proof.  There is no such baseline to work from, and
+  it would set the bar far too high.  On the other hand, for
+  proposals involving syntactic changes, it is very reasonable to ask for
+  a BNF for the changes.
+
+  Ultimately, the necessary degree of precision is a judgement that the committee
+  must make; but authors should try hard to offer precision.
+
+* It should offer evidence of utility.  Even the strongest proposals carry costs:
+
+  * For programmers: they usually make the language just a bit more complicated;
+  * For GHC maintainers: they make the implementation a bit more complicated;
+  * For those wanting to proposals in the future: they usually add new back-compat burdens, and consume syntactic design space.
+  * Moreover, all these costs contitute a permanent tax on every future programmer, langauge designer, and GHC maintainer.
+
+  The tax may well be worth it (a language without polymorphism
+  would be simpler but we don't want it), but the case should be made.
+
+  The case is stronger if lots of people jump in saying "yes, this would be so
+  useful to me".  The committee is often faced with proposals that are reasonable,
```

Comment:
Do we really want to invite this noise on the github threads? I liked the earlier suggestion we had about named supporters in the proposal itself.

Reply:
There are message reactions in Github like +1 and -1 made specifically to allow expressing "this would be useful to me" without polluting discussion at the same time. Not sure if it's okay to depend on Github-specific features, tho.

Reply:
We built the process around GitHub delibarately, so yes, we can do that.

Reply:
Shall we suggest that people just thumbs-up a proposal if they like it, or comment if they have some more substantive evidence?

Reply:
How about this: we invite people to thumbs-up if they like it, but to post if they have a fresh example to share. Or, if their example is proprietary, they can just say that Company X has an example that would benefit from this. I find new examples very, very helpful.